pkgs/development/python-modules/freertos-gdb/default.nix

```
@@ -6,11 +6,11 @@
 
 buildPythonPackage rec {
   pname = "freertos-gdb";
-  version = "1.0.3";
+  version = "1.0.4";
 
   src = fetchPypi {
```

```suggestion
  version = "1.0.4";
  pyproject = true;

  src = fetchPypi {
```
and specify `build-system`.